cfgov/v1/models/banners.py

```diff
@@ -0,0 +1,38 @@
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils.safestring import mark_safe
+
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.core.fields import StreamField
+
+from v1.atomic_elements.molecules import Notification
+
+
+class Banner(models.Model):
+    title = models.CharField(
+        max_length=255,
+        help_text='For internal reference only; does not appear on the site.'
+    )
+    url_patterns = models.CharField(
```

Comment:
Nitpicky, but why is this plural? Shouldn't it be `url_pattern`?

Reply:
Yeah, I suppose you're right! Will update.

Reply:
Amended original commit with this change, now in fc9e180.